frontend/package.json

```diff
@@ -21,6 +22,7 @@
     "react-scripts": "5.0.0",
     "react-table": "^7.7.0",
     "react-toastify": "^8.1.1",
+    "swr": "^1.2.2",
```

Comment:
And we definitely don't want to take on `swr` again.  We've pivoted from `swr` and `isomorphic-unfetch` to using `useBackend` (based on React Query Library) and `axios`.

Please refactor the code so that we don't have to take on more needless dependencies.

<img width="425" alt="image" src="https://user-images.githubusercontent.com/1119017/157350780-4de5ccd3-60d3-486e-b440-829f4d8b04bd.png">